app/screens/splash/splash.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import { 
+  View, 
+  Text,
+  Image,  
+  TextInput,
+ } from 'react-native';
+import TimerMixin from 'react-timer-mixin';
+ 
+import style from './style';
+
+// import 
+ class SplashScreen extends Component {
+
+  mixins = [TimerMixin];
```

Comment:
It is generally recommended to not use mixins. Timer functions work fine without the need of timer mixins. I'm not sure if they are deprecated but are definitely considered harmful. https://reactjs.org/blog/2016/07/13/mixins-considered-harmful.html

Reply:
🆗